checkout-extension/README.md.liquid

```diff
@@ -2,87 +2,38 @@
 
 Checkout UI extensions let app developers build custom functionality that merchants can install at defined targets in the checkout flow. You can learn more about checkout UI extensions in Shopify’s [developer documentation](https://shopify.dev/api/checkout-extensions/checkout).
 
-
 ## Prerequisites
-Make sure to complete [all the prerequisites](https://shopify.dev/apps/checkout/delivery-instructions/getting-started#requirements) from any tutorials to develop a Checkout UI extension.
```

Comment:
The old README was quite long. I tried to narrow this down to just the most critical concepts for a developer to understand.